pkgs/by-name/nh/nh/package.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, rustPlatform
+, installShellFiles
+, makeWrapper
+, fetchFromGitHub
+, nvd
+, use-nom ? true
+, nix-output-monitor ? null
+}:
+
+assert use-nom -> nix-output-monitor != null;
+
+let
+  version = "3.4.12";
+  runtimeDeps = [ nvd ] ++ lib.optionals use-nom [ nix-output-monitor ];
+in
+rustPlatform.buildRustPackage {
+  inherit version;
+  pname = "nh";
+
+  src = fetchFromGitHub {
+    owner = "ViperML";
+    repo = "nh";
+    rev = version;
+    hash = "sha256-V5TQ/1loQnegDjfLh61DxBWEQZivYEBq2kQpT0fn2cQ=";
+  };
```

Comment:
Seems as if you messed up the `rev`.

I think best would be `refs/tags/v${version}`.

```text
source> building '/nix/store/jyqhkp4fdv8qa43ghkb3059c6v8cswb8-source.drv'
source>
source> trying https://github.com/ViperML/nh/archive/3.4.12.tar.gz
source>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
source>                                  Dload  Upload   Total   Spent    Left  Speed
source>   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
source>   0    14    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
source> curl: (22) The requested URL returned error: 404
source> error: cannot download source from any mirror
```

Reply:
Oops. Going to fix that. Thx !